src/routes/data-decode/data-decoded.service.spec.ts

```diff
@@ -32,30 +32,6 @@ describe('DataDecoded Service', () => {
     );
     expect(mockDataDecodedRepository.getDataDecoded).toBeCalledTimes(1);
   });
-
-  it('should throw an error on invalid payload', async () => {
-    const chainId = faker.datatype.string();
-    const createDataDecodedDto = createDataDecodedDtoFactory(
-      undefined,
-      faker.random.word(),
-    );
-
-    await expect(
-      service.getDataDecoded(chainId, createDataDecodedDto),
-    ).rejects.toThrow('Invalid payload');
-  });
-
-  it('should throw an error on invalid payload (2)', async () => {
-    const chainId = faker.datatype.string();
-    const createDataDecodedDto = createDataDecodedDtoFactory(
-      faker.random.words(),
-      faker.random.word(),
-    );
-
-    await expect(
-      service.getDataDecoded(chainId, createDataDecodedDto),
-    ).rejects.toThrow('Invalid payload');
-  });
 });
```

Comment:
Moving validation checks to the e2e tests since this validation step is also moved from the service to the controller/validation pipe.

Reply:
Do you think this test class should exist now? I would say that it is a single test that should be already covered by the controller (and this one is tied to the implementation)

Reply:
Yes, I agree, this should be covered already, I've deleted it in [3a1de96](https://github.com/5afe/safe-client-gateway-nest/pull/278/commits/3a1de96b74e40438d80e3a094f6e635d7330182d)

Reply:
I've created another PR to delete some other test classes whose test cases should be already covered by controller layer unit tests: https://github.com/5afe/safe-client-gateway-nest/pull/284 🙂